terraform/gitlab/ci-templates/k8s-cluster/.gitlab/scripts/pushtorepo.sh

```diff
@@ -14,6 +14,6 @@ cp -r $SRC_DIR/. $DEST_DIR_NAME/
 git config --global user.email "root@${gitlab_hostname}"
 git config --global user.name "root"
 git add $DEST_DIR_NAME/.
-git commit -m "[skip-ci]update generated configs to project"
+git commit -m "update generated configs to project"
```

Comment:
remove skip-ci, so that deploy can be triggered (if this is the last commit) in case of changes in profiles